Makefile

```diff
@@ -37,8 +41,59 @@ ut-circle: binary
 	--with-xunit --xunit-file=/circle_output/output.xml; RC=$$?;\
 	[[ ! -z "$$COVERALLS_REPO_TOKEN" ]] && coveralls || true; exit $$RC'
 
+dist/calicoctl:
+	mkdir -p dist
+	curl -L http://www.projectcalico.org/latest/calicoctl -o dist/calicoctl
+	chmod +x dist/calicoctl
+
+run-kubernetes-master: run-etcd run-service-proxy binary dist/calicoctl kubectl
+	# Stop any existing kubelet that we started
+	-docker rm -f calico-kubelet-master
+
+	# Remove any pods that the old kubelet may have started.
+	-docker rm -f $$(docker ps | grep k8s_ | awk '{print $$1}')
+
+	# Run the kubelet which will launch the master components in a pod.
+	docker run \
+	--name calico-kubelet-master \
+	--volume=`pwd`/dist:/usr/libexec/kubernetes/kubelet-plugins/net/exec/calico:ro \
+	--volume=/:/rootfs:ro \
+	--volume=/sys:/sys:ro \
+	--volume=/dev:/dev \
+	--volume=/var/lib/docker/:/var/lib/docker:rw \
+	--volume=/var/lib/kubelet/:/var/lib/kubelet:rw \
+	--volume=/var/run:/var/run:rw \
+	--net=host \
+	--pid=host \
+	--privileged=true \
+	-e KUBE_API_ROOT=http://localhost:8080/api/v1/ \
+	-d \
+	gcr.io/google_containers/hyperkube:v$(K8S_VERSION) \
+	/hyperkube kubelet --network-plugin calico --v=5 --containerized --hostname-override="127.0.0.1" --address="0.0.0.0" --api-servers=http://localhost:8080 --config=/etc/kubernetes/manifests
+
+	# Start the calico node
+	sudo dist/calicoctl node
+
+run-service-proxy:
+	-docker rm -f calico-service-proxy
```

Comment:
While it acts as a service proxy, it is always referred to as the "kube-proxy", so I'd keep that naming to avoid confusion.